blockchains/smartchain/assets/0xba7E2a9E5193E60368F440E4Ae881cC312d6a160/info.json

```diff
@@ -0,0 +1,36 @@
+{
+    "name": "Minions INU",
+    "type": "BEP20",
+    "symbol": "MINION",
+    "decimals": 9,
+    "website": "https://minionsinu.net/",
+    "description": "MinionsINU is designed to create a complex ecosystem in which DeFi and Metaverse are integrated together. Crypto users will access all DeFi & Metaverse services quickly, cheaply and effectively with only MinionsINU platforms. ",
+    "explorer": "https://bscscan.com/token/0xba7e2a9e5193e60368f440e4ae881cc312d6a160",
+    "status": "active",
+    "id": "0xba7E2a9E5193E60368F440E4Ae881cC312d6a160",
+    "links": [
+        {
+            "name": "twitter",
+            "url": "https://twitter.com/MinionsINUbsc"
+        },
+        {
+            "name": "telegram",
+            "url": "https://t.me/MinionsINUbsc"
+        },
+        {
+            "name": "youtube",
+            "url": "https://www.youtube.com/@minionsinu"
```

Comment:
you need remove ```www```

Reply:
okey i ll try to remove it

Reply:
I try to delete but i guess there is a mistake. I ll recreate new request for fix it.